redmine/redmine-connector/redmine-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/redmine-versions/getVersions.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~  Copyright (c) 2014-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~
+~  WSO2 Inc. licenses this file to you under the Apache License,
+~  Version 2.0 (the "License"); you may not use this file except
+~  in compliance with the License.
+~  You may obtain a copy of the License at
+~
+~    http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing,
+~ software distributed under the License is distributed on an
+~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+~ KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations
+~ under the License.
+-->
+<template name="getVersions" xmlns="http://ws.apache.org/ns/synapse">
+  <parameter name="id"/>
+  <sequence>
+    <property expression="$func:id" name="uri.var.id" scope="default" type="STRING"/>
+    <call>
+      <endpoint>
+        <http method="get" trace="disable" uri-template="{+uri.var.apiUrl}/projects/{+uri.var.id}/versions.{+uri.var.responseType}"/>
```

Comment:
This won't log the detailed trace information including message payloads. So it's better if you remove trace="disable" from this endpoint.